geniza/corpus/models.py

```diff
@@ -221,8 +221,11 @@ class DocumentType(models.Model):
     """The category of document in question."""
 
     name = models.CharField(max_length=255, unique=True)
+    display_label = models.CharField(max_length=255, blank=True)
```

Comment:
Thanks! How do we handle translations for help text, by the way?

Reply:
We need to start tagging them! I was just reading more about django localization because I had a question about how to test it, and I saw that they recommend you explicitly set model verbose names and make them translatable — which we have _not_ done yet. I think hold off on making this one translatable and I'll add a chore to set it up across all the models (verbose names, help text, etc.)